crates/benches/native_vps.rs

```diff
@@ -60,6 +56,10 @@ use namada_node::bench_utils::{
     TX_BRIDGE_POOL_WASM, TX_IBC_WASM, TX_INIT_PROPOSAL_WASM, TX_RESIGN_STEWARD,
     TX_TRANSFER_WASM, TX_UPDATE_STEWARD_COMMISSION, TX_VOTE_PROPOSAL_WASM,
 };
+use namada_token::masp::{
```

Comment:
let's import this from apps_lib - it re-exports all of sdk
```suggestion
use namada_apps_lib::token::masp::{
```